doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,608 @@
+The NEST documentation style guide
+==================================
+
+.. contents:: On this page, you'll find
+   :local:
+
+Why do we have a style guide?
+-----------------------------
+
+This style guide was created to provide a single reference point for content
+creation in NEST. This helps ensure the NEST user-level documentation remains
+clear and consistent. The style choices we make here are meant to follow the
+major trends in technical writing for software.
+
+Contribute to the docs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to add or modify documentation, make sure to read through this guide before writing a contribution.
+
+We have templates for :doc:`Python example scripts <pyapi_template>` and the :doc:`PyNEST API docstrings <example_template>`.
+
+You will also need to know how to build the documentation locally on your machine: See
+:doc:`../../documentation_workflow/user_documentation_workflow`.
+
+For additional information, including the Git workflow and making a pull request see :doc:`../index`
+
+
+General guidelines about writing for NEST
+-----------------------------------------
+
+A few tips on writing documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Be concise: Use short words and sentences; avoid unnecessary modifiers
+
+- Be consistent: Follow this style guide and relevant templates
+
+- Be specific: Avoid vague language; clarity is key
+
+- Focus your message: Start with what's most important, and keep text relevant
+  within given section
+
+- Write for your reader: Try to understand what your reader needs to know;
+  include context to what you're saying. Remember that most NEST users are computational neuroscientists,
+  including students, not software developers!
+
+- Provide examples: Show, don't tell; if you can use code snippets, screenshots,
+  or figures to illustrate a point, do so!
+
+- Avoid terms like "simply", "just", or "easy": your knowledge does not always equal
+  your readers.  Don't make your reader feel bad because they "simply" didn't
+  understand something.
+
+- Write positively: Use positive language rather than negative language
+
+Remember that the primary purpose of the documentation is to provide
+information to NEST users who are looking for it.
+
+Voice
+~~~~~~
+
+Voice is *how* we talk to people through writing. The voice encompasses
+substance, style, and tone.
+
+Our voice is
+
+- empathetic: we write thinking of who we are writing for and what their needs
```

Comment:
Not rendered as list. Periods missing at end.

Reply:
Periods added